Modules/Office365DSC/Modules/O365DSCReverse.psm1

```diff
@@ -1011,95 +1011,19 @@ function Start-O365ConfigurationExtract
     }
     #endregion
 
-    #region "SCFilePlanPropertyAuthority"
```

Comment:
It may be due to a merging conflict somehow, but why are you removing the logic for the FilePlanProperties?